analyzers/its/sources/BlazorSample/BlazorSample/Pages/S6802/S6802.razor

```diff
@@ -0,0 +1,73 @@
+﻿@foreach (var button in Buttons)
```

Comment:
Another question here: does it make sense to have these files here even if we are not expecting issues to be raised?

Reply:
As mentioned by @antonioaversa in his [comment](https://github.com/SonarSource/sonar-dotnet/pull/8341#discussion_r1389180283):
> The day this bug on the Microsoft side is fixed, these ITs will break, since the actual will contain more issues than expected. I think this is a good thing because we want to be notified of the impact that an external change is going to have on our rule.

I will keep these tests in .NET ITs but reduce the content.